src/components/NewLocationPage/CountyMap/CountyMap.style.tsx

```diff
@@ -12,7 +12,7 @@ export const PinnedContainer = css`
   z-index: 901;
 `;
 
-export const MapContainer = styled.div`
+export const MapContainer = styled.div<{ offsetAmount: number }>`
```

Comment:
Consider tweaking this name since:
1. "offsetAmount" isn't very clear.  What is the offset relative too?  Is it an x offset or y offset or something else?  TBH this isn't really an offset.  It's just a fixed position for the bottom.
2. "Amount" doesn't really add any information and could probably be dropped."

Perhaps `bottomPosition` or `fixedBottomPosition` or something.

I'd also be tempted to make the type `number | null` instead of treating `0` as a special value that sets it to `none`.  I think using `null` for that would be cleaner.